assets/wizards/popups/components/segmentation-preview/index.js

```diff
@@ -28,6 +33,8 @@ const SegmentationPreview = props => {
 	const decorateURL = urlToDecorate => {
 		const view_as = segment.length ? [ `segment:${ segment }` ] : [ 'segment:everyone' ];
 
+		view_as.push( `cid:${ uniqueId( 'view_as_client_' ) }` );
```

Comment:
Seems like there's something off with the `cid` URL param - the previewing navigation is still treated as a series of separate sessions. Maybe this `cid` value is recreated on navigation events?

Reply:
Ah yes, because we run `decorateUrl` on every anchor link in the preview window, and the spoofed `cid` was getting created as part of that function. Moving the `cid` creation outside of the function fixes it in 34eb565b.

Reply:
Pointed out an issue with client ID spoofing here: https://github.com/Automattic/newspack-popups/pull/451#discussion_r583494755

Reply:
Removed all client ID spoofing from this PR and https://github.com/Automattic/newspack-popups/pull/451 for now. Let's handle that in #891 and https://github.com/Automattic/newspack-popups/pull/458.